src/algorithms/expected_consensus.md

```diff
@@ -292,10 +339,16 @@ w[r+1] = w[r] + floor(1000 * (wForkFactor(wPowerFactor[r+1] + wBlocksFactor[r+1]
 
```

Comment:
@Kubuxu, section of interest for you is below.

Reply:
I would be for defining function: `log2b(x) = floor(log2(x))` and explaining that it is a binary length of `x`.

Reply:
In general, any division should be left for last (as it is the lossy operation in integer arithmetic). The moment division is performed should be defiend using the paranthesis so there is no ambiuity.

 The multiplication by 1000 should be done in every term before the division is performed to achieve greater precision. (another option could be multiplication by 2^16 as it maps well to bigint arithmetic).

Reply:
I would also move the note about integer arithmetic from the bottom to the top.